searchpage.html

```diff
@@ -1,37 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link rel="icon" type="image/x-icon" href="./assets/images/favicon.ico">
-    <link
-      href="https://fonts.googleapis.com/css2?family=Corinthia&family=Josefin+Slab:wght@100&family=Licorice&family=Montserrat:wght@500&family=Vujahday+Script&display=swap"
-      rel="stylesheet"
-    />
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Name of Project</title>
     <link
       rel="stylesheet"
       href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
       integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
       crossorigin="anonymous"
     />
-    <title>Name of Project</title>
-    <link rel="stylesheet" href="./assets/css/searchpage.css" />
-  </head>
+    <link rel="stylesheet" href="./assets/css/searchpage.css">
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/handlebars/4.7.7/handlebars.min.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link href="./assets/css/locatorplus.css" rel="stylesheet">
+    <script src="./assets/js/locatorplus.js"></script>
+    <script>
+      const CONFIGURATION = {
```

Comment:
This is what centers the map around a certain area right? 

I am wondering if this is necessary? It looks like I can't search for any other locations (for example I live in Federal Way and when I search for Federal Way I get defaulted to this constant address).

Reply:
I've added more locations to the list but it's address near Seattle, I'm wondering if there's a way to populate this by using the kroger API or I will just have to manually add them.